GeeksCoreLibrary/Modules/GclReplacements/Services/ReplacementsMediator.cs

```diff
@@ -407,6 +407,7 @@ public string HandleVariablesDefaultValues(string input, string prefix = "{", st
             return input;
         }
 
+        // This regex will match any variable that has a default value. Except for several special cases that are used in baskets, that use a similar syntax.
```

Comment:
As discussed, comment is no longer needed since functionality changed.

Reply:
I removed the comment.